addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1716,6 +1716,7 @@ export class OdooEditor extends EventTarget {
      * @returns {?}
      */
     _applyCommand(...args) {
+        this._checkStepUnbreakable = true;
```

Comment:
Forcing it here seems to defeat the purpose of the variable though. Its definition seems to be `Wether we should check for unbreakable the next history step.` so ok we can bend it to exclude commands and say it's only when doing history step outside of a proper command, but then we should restore the previous value that was set before the command IMHO.

Reply:
I didn't go with that logic in the begining , because we alreading override `_checkStepUnbreakable` in https://github.com/odoo/odoo/blob/15.0/addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js#L801  so this may result in some unexpected behavior if we reset it to the old value , what do you think ?

Reply:
Ok I didn't notice that. Then your fix looks correct. Thanks!